org.v1993.NXDumpClient.yml

```diff
@@ -0,0 +1,34 @@
+app-id: org.v1993.NXDumpClient
```

Comment:
v1993.org doesn't seem to resolve, it has to be a domain under the control of the project/developer

Reply:
I've mentioned this in description. I suppose I can acquire in now if it's a hard requirement, although I'd prefer to be sure that it is first.

Reply:
Yes it is a blocker

Reply:
You can always use `io.github.foo.Foo` if you don't want to acquire the domain

Reply:
**one doesn't simply invent a domain for their app-id.**

use a github associated app-id.

Reply:
> I've mentioned this in description. I suppose I can acquire in now if it's a hard requirement, although I'd prefer to be sure that it is first.

you don't need to buy a domain. Just don't use one you don't own or for which there is no delegation. That's why you can use `io.github`.

Reply:
> **one doesn't simply invent a domain for their app-id.**

Unless one does :upside_down_face:. I acquired ownership of v1993.org and put up a stub page there using GitHub pages. Is this sufficient for now?

Reply:
Yes